snomed-ct-integration/pom.xml

```diff
@@ -73,6 +68,12 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-slf4j2-impl</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Why did you include a dependency on the starter data builder?

Reply:
oh no, this was a mistake. deleted!